src/app/shared/components/notification-popup-container/notification-popup-container.component.scss

```diff
@@ -9,5 +9,5 @@
   flex-direction: column;
   gap: 8px;
 
-  z-index: 300;
+  z-index: 1001;
```

Comment:
What was covering it?

Reply:
Dialog window 
![Screenshot 2024-04-11 101632](https://github.com/tv2/ng-sofie/assets/152977713/345c8493-3209-4e35-8963-f9cd2c743707)